packages/babel-preset-pwa-kit/index.js

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+module.exports = function() {
+    return {
+        presets: [
+            [
+                require('@babel/preset-env'),
+                {
+                    targets: {
+                        node: 12
```

Comment:
node 12 is end-of-life at the end of the week. Do we still want to target it or should we bump to 14?

Reply:
Good call out. It would be nice to keep that change in a separate pr. This one is getting a little large as it is.